app/graphql/inputs.js

```diff
@@ -5,8 +5,8 @@ module.exports = gql`
     firstName: String!
     lastName: String!
     username: String!
-    email: String!
-    password: String!
+    email: String! @constraint(format: "email", minLength: 12, pattern: ".*@wolox.(ar|co|cl|com.ar)$")
+    password: String! @constraint(minLength: 8)
```

Comment:
password should be alphanumeric, it should throw error for special characters as: `_*'?`

Reply:
I forgot it